.github/workflows/release.yml

```diff
@@ -124,8 +124,24 @@ jobs:
           source $GITHUB_WORKSPACE/scripts/ci/print_logs.sh
         if: failure() # only run this job if the build step failed
 
-  Publish-Release:
+  Test-Release-Artifacts:
     needs: Build-Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download release artifacts
+        uses: actions/download-artifact@v2
+        with:
+          name: tiledb-dist
+          path: dist
+      - name: Test names of release artifacts
+        run: |
+          if [ ls dist/ | grep -Ev -- '^tiledb-(linux|macos|windows)+-(x86_64|arm64)(-noavx2)?-.+-[0-9a-f]{7}\.(zip|tar\.gz)$' ]; then
```

Comment:
This `grep` invocation took quite some time to write. The regex was fine but I had to pass `-E` to recognize the `+` quantifier, and `--` to recognize the dashes. 😅